src/metadata/metadata_superblock.c

```diff
@@ -438,6 +438,8 @@ struct ocf_metadata_superblock
 {
 	struct ocf_metadata_segment segment;
 	struct ocf_superblock_config *config;
+	struct ocf_metadata_raw *raw;
+	ocf_cache_t cache;
```

Comment:
Why do we need those? The `ocf_metadata_sb_zero()` could just take those as args, as those are available in the call context anyway.

Reply:
to make it a self contained object